### PR TITLE
7.0 binary field

### PR DESCRIPTION
**Pull request by sebastienbeau**
Here is a proposal too add new fields for image and binary. There is a example module so you can try it. But before merging I propose to remove it.

This merge was before on lp https://code.launchpad.net/~akretion-team/server-env-tools/server-env-tools/+merge/222291

### CODE REVIEW

File: binary_field/< [hunk 1]
```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
```
Comment (bwrsandman) [inline binary_field/<]: I think this file might be mis-named

**Comment by sebastienbeau** [comment]
Hi Sandy, sorry a wrong file have been added, and I forget to remove it when moving the branch to github. It's done now. Thanks

**Comment by tmotyl** [comment]
why this pull request was closed? I can not see the code from it neither in v7 nor v8 branch.

**Comment by michaelkarrer81** [comment]
+1

**Comment by bwrsandman** [comment]
The PR was closed probably due to the removal of the master branch which is now 8.0

**Pull request closed without merging**